std/net/curl.d

```diff
@@ -801,6 +801,7 @@ if (isCurlConn!Conn)
                                 text("No filename specified to delete for URL ", url));
         conn.addCommand("DELE " ~ t[1]);
         conn.perform();
+	return "DELE" ~ t[1];
```

Comment:
I think it's better to return `null` rather than dummy data. Also, let's fix the indentation.

Testing the new "suggestions" feature:

```suggestion
        return null;
```